context/app/static/js/Routes.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import NoticeAlert from './components/NoticeAlert';
+import Details from './components/Details';
+import Home from './components/Home';
+import Search from './components/Search/Search';
+
+function Routes(props) {
+  const { flaskData } = props;
+  const {
+    flashed_messages, entity, provenance, vitessce_conf,
```

Comment:
I'd say these should be renamed on assignment to be consistent with JS naming conventions? But not 100% sure it's worth the trouble, either. Thoughts?

Reply:
I'll create an issue and revisit.

Reply:
Created #258